tests/testthat/test-4IO.R

```diff
@@ -11,6 +11,116 @@ test_that("Importing biom files yield SummarizedExperiment objects", {
     expect_s4_class(me2, "SummarizedExperiment")
     expect_equal(dim(me), dim(me2))
     expect_equal(rowData(me), rowData(me2))
+    
+    biom_object <- biomformat::read_biom(
+        system.file("extdata/testdata/Aggregated_humanization2.biom",
```

Comment:
This file is correct place for tests --> moved them here

Reply:
Sorry, I didn't understand the comment I didn't touch that file :)